src/kit.ts

```diff
@@ -299,8 +299,9 @@ export async function vsInstallations(): Promise<VSInstallation[]> {
   const installs = [] as VSInstallation[];
   const inst_ids = [] as string[];
   const vswhere_exe = path.join(thisExtensionPath(), 'res/vswhere.exe');
-  const vswhere_args = ['-all', '-format', 'json', '-products', '*', '-legacy', '-prerelease'];
-  const vswhere_res = await proc.execute(vswhere_exe, vswhere_args, null, {silent: true, encoding: 'utf8'}).result;
+  const vswhere_args =
+      ['/c', 'chcp 65001 |', vswhere_exe, '-all', '-format', 'json', '-products', '*', '-legacy', '-prerelease'];
```

Comment:
Does this work? I'd think `child_process` would quote the arguments before calling CreateProcess and we'd end up with something like

```bash
cmd.exe /c "chcp 65001 |" vswhere.exe -all -format -json -products * -legacy -prerelease
```

That'll behave weird unless cmd.exe is doing its own goobery command-line parsing and not using `argv` from the CRT

Reply:
Just tested, and can confirm that cmd.exe is doing something goobery:

```bash 
> cmd.exe /c "echo hi" echo foo
hi echo foo
```

Reply:
hm, it does work for me